### ISSUE TITLE
What's the differences between this paper and "fBGD: Learning Embeddings From Positive Unlabeled Data with BGD"

### ISSUE THREAD

**Issue by chenchongthu** [XinGla/AllVec#1]
It seems that the two papers are very similar except for different tasks.

**Comment by ghost** [comment]
The difference is that fBGD using gradient descent to optimize the loss function. However, this paper provides a framework and the optimization can be done by first order gradient or second order newton methods. The specific design for different tasks also differs.
